src/utils/compiler.ts

```diff
@@ -3,7 +3,18 @@ import pick from 'lodash.pick'
 import * as ProcessType from 'mesg-js/lib/api/typedef/process'
 import {hash, Process, Service} from 'mesg-js/lib/api/types'
 
-const decode = (content: Buffer) => yaml.safeLoad(content.toString())
+const replaceVariable = (value: any, env: { [key: string]: string }) => {
+  if (typeof value !== 'string') return value
+  const reg = new RegExp('\\$\\(env\\:(.*)\\)', 'g')
+  return value.replace(reg, (match: string, p1: string) => env[p1])
```

Comment:
Should throw error if env not set:
```suggestion
  return value.replace(reg, (match: string, p1: string) => {
    if (!Object.keys(env).includes(p1)) {
      throw new Error('env variable ' + p1 + ' must be set')
    }
    return env[p1]
  })
```